tips/migrations/0001_initial.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.0.3 on 2022-04-14 08:58
```

Comment:
You already have this file in https://github.com/redhat-beyond/Xpense/pull/49 PR, 
why you created it again?